### DISCUSSION TITLE
Can we start to organize the project tab?

### DISCUSSION THREAD

**Discussion by bhack** [openxla/community#32] [stablehlo]
It will be very useful to understand and track where the project it is going, at an high level, if we can start to use the [Project tab](https://github.com/openxla/stablehlo/projects).

The issues are already labeled quite well so we are already in a good position to organize/automatic the Kannan boards in the Project tab.

E.g. see the nice org at https://github.com/orgs/iree-org/projects/1/views/1

/cc @burmako

**Comment by burmako** [comment]
Hi @bhack! Within StableHLO, we've been recently organizing issues into projects similarly to what you proposed, and earlier today we tied it all together in a roadmap: https://github.com/openxla/stablehlo/blob/main/docs/roadmap.md. 

This is just an initial version - I expect that the organization and the taxonomy of the workstreams will evolve over time, but I hope that this already provides useful structure behind almost 300 tickets in the StableHLO issue tracker. And, of course, feedback is very welcome!

**Comment by bhack** [comment]
Thanks, it is very useful. I hope OpenXLA could adopt these OSS accessibility best practices across all repositories as a standard.

**Comment by bhack** [comment]
@burmako If I enable the linked PR field in your view I see that probably we are not tracking the issue status as we have isse with already linked PR without a status.  I have expected to have these in Progress, Under review, etc..
What is the best way to maintain a realistic status progress without too much recurrent misalignment?
